### ISSUE TITLE
test/buildbot/builders/unified_cmakeex.py fails at least since 'hint' was added

### ISSUE THREAD

**Issue by asb** [llvm/llvm-zorg#235]
I had a quick look at fixing this but I wasn't completely sure what the current `partly_rendered` logic is trying to achieve.

This is the error, which seems to stem from the format string being passed to `util.Interpolate` now using syntax that isn't really compatible with Python `%` based formatting.

```
Traceback (most recent call last):
  File "/home/asb/repos/llvm-zorg/test/buildbot/builders/unified_cmakeex.py", line 93, in <module>
    assert factory_has_step(f, "Checkout the source code")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/asb/repos/llvm-zorg/test/buildbot/builders/unified_cmakeex.py", line 38, in factory_has_step
    v = partly_rendered(s.kwargs.get("name"))
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/asb/repos/llvm-zorg/test/buildbot/builders/unified_cmakeex.py", line 19, in partly_rendered
    return r.fmtstring % interpolations
           ~~~~~~~~~~~~^~~~~~~~~~~~~~~~
KeyError: 'kw:hint:-'
```

**Comment by vvereschaka** [comment]
Thank you @asb ,
I'll update the tests properly.

**Comment by vvereschaka** [comment]
Fixed by #237
